src/System.Device.Gpio/System/Device/Gpio/Drivers/LibgpiodDriver.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    public class LibgpiodDriver : GpioDriver
+    {
+        private const string GpioGet = "gpioget";
+        private const string GpioChip0 = "gpiochip0 ";
+        private const string GpioSet = "gpioset";
+        public int _waitMilliSeconds { get; set;}
+        private Dictionary<int, PinMode> _pinNumperToPinModes;
+        private Dictionary<int, Process> _pinNumperToProcess;
```

Comment:
Instead of having two dictionaries, does it make more sense to have a dictionary like `Dictionary<int, ValueTuple<PinMode, Process>>`

Reply:
yeah that makes sense